### PR TITLE
[dhcp_server] add show dhcp server info

### PR DESCRIPTION
**Pull request by Xichen96**
#### Why I did it
Add show dhcp_server info

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Add new subcommand to show dhcp_server command.

#### How to verify it
Add unit tests for compile time testing. Manually run command in latest master image.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)
Latest master image

- [ ] 
- [ ] 

#### Description for the changelog

Add show dhcp_server info

### CODE REVIEW

File: dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py [hunk 1]
```diff
@@ -79,5 +79,26 @@ def range(db, range_name):
...
 
 
+@ipv4.command()
+@click.argument('dhcp_interface', required=False)
+@click.option('--with_customized_options', default=False, is_flag=True)
+@clicommon.pass_db
+def info(db, dhcp_interface, with_customized_options):
+    if not dhcp_interface:
+        dhcp_interface = "*"
+    headers = ["Interface", "Mode", "Gateway", "Netmask", "Lease Time(s)", "IP Bind"]
```
Comment (yaqiangz) [inline dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py:89]: Last header should be "State"

Comment (Xichen96) [reply dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py]: @yaqiangz updated

**Review by yaqiangz — APPROVED** [review]
LGTM

**Comment by Xichen96** [comment]
@yxieca Checks have now passed. Could you help review?

**Pull request merged**